### ISSUE TITLE
Thumbnails too small on larger screens (Fairphone 2, Nexus 5)

### ISSUE THREAD

**Issue by sam-hobbs** [accumulator/Quickddit#16]
Hi,

This is one of my favourite Sailfish apps, thanks for all the hard work on it.

I think I read in a changelog a while back that you had done some work to make the UI scalable, which is great if I'm not misremembering. Is it possible to tweak the thumbnails so that they are larger on bigger screens? They are too small to be easily visible on Fairphone 2 (1290x1080 5" screen), but the same feed viewed on Jolla looks great.

Here's Quickddit on Fairphone 2 (ithumbnails too small):
![screenshot-16-05-10-20-43-26 - quickddit](https://cloud.githubusercontent.com/assets/7817338/15161119/d526a35e-16f4-11e6-9aa4-93922f660702.png)

And here's Neliapila on Fairphone 2 (thumbnails easily visible):
![screenshot-16-05-10-20-41-47 - neliapila](https://cloud.githubusercontent.com/assets/7817338/15161122/d909b5c4-16f4-11e6-9a72-e9f3108c1428.png)

Something closer to the Neliapila scale would be great, if that's possible?

**Comment by accumulator** [comment]
can you download the latest from openrepos, start quickddit from the terminal, and report what PPI is detected?

Thanks.

**Comment by sam-hobbs** [comment]
Test with latest version:

[code][nemo@localhost ~]$ harbour-quickddit
[D] QWaylandEglClientBufferIntegration::QWaylandEglClientBufferInte
gration:62 - Using Wayland-EGL
[D] QMLUtils::setPScale:212 - Device PPI = 100 scale =  1
[W] DeclarativeWindow::DeclarativeWindow:93 - No default allowed or
ientations defined. Check your device configuration![/code]

I remember the changes you made in response to that comment on openrepos made a difference, but I think I responded before I tested it fully (got excited by the changelog!). I don't have the N5 anymore but I'll ask someone to do the test on TMO and post it here for you, and attach a screenshot if possible.
